### PR TITLE
Update a dead link to Jenkins builds.

### PR DESCRIPTION
**Pull request by peihe**

### CODE REVIEW

**Comment by peihe** [comment]
retest this please

File: src/contribute/testing.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ precommit and postcommit testing.
 ### Precommit
 
 For precommit testing, Beam uses
-[Jenkins](https://builds.apache.org/view/Beam/) and a code coverage tool
+[Jenkins](https://builds.apache.org/) and a code coverage tool
```
Comment (aaltay) [inline src/contribute/testing.md]: Should we change this to `https://builds.apache.org/view/A-D/view/Beam/` instead?

Comment (peihe) [inline src/contribute/testing.md]: This is better, updated.

**Comment by peihe** [comment]
R: @aaltay, @kennknowles

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/608/

Jenkins built the site at commit id d11ffef2e56a9dd70d060e5bcdcf1c1c67a071d8 with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/283/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by aaltay** [comment]
LGTM. Thank you.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/609/

Jenkins built the site at commit id b2b8a19d689280b8bd265f32b9d5457367c8396c with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/283/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by jkff** [comment]
@asfgit merge

**Pull request closed without merging**